templates/windows2k16.tpl.yaml

```diff
@@ -68,7 +68,7 @@ objects:
             "valid": "jsonpath::.spec.domain.devices.disks[*].disk.bus",
             "rule": "enum",
             "message": "disk bus has to be either virtio or sata",
```

Comment:
The rule message should be updated as well

Reply:
fixed